npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -44,7 +44,15 @@ class VersionResolver
         # what ts-jest requests\n
         YARN_BERRY_PEER_DEP_ERROR_REGEX =
           /
-            YN0060:\s|\s.+\sprovides\s(?<required_dep>.+?)\s\((?<info_hash>\w+)\).+what\s(?<requiring_dep>.+?)\srequests
+            YN0060:\s│\s.+\sprovides\s(?<required_dep>.+?)\s\((?<info_hash>\w+)\).+what\s(?<requiring_dep>.+?)\srequests
```

Comment:
Great eye catching the incorrect symbol here! How did you find about it? I wonder if we should remove the symbol from the regexp and relax it a bit instead, to avoid this kind of confusion (and to potentially avoid breakage if yarn decides to format errors using a different ASCII char or something). I think removing `\s│` from the regexp altogether may work.

Reply:
Yeah, removing the `\s│` makes sense as the error code matching in the beginning ensures we're dealing with the intended error string.